bellows/ezsp.py

```diff
@@ -31,17 +32,31 @@ def connect(self, device):
     def reset(self):
         return self._gw.reset()
 
+    @asyncio.coroutine
+    def version(self, version):
+        result = yield from self._command('version', version)
+        if result[0] != version:
+            LOGGER.debug("Switching to version %d", result[0])
+            result = self._command('version', result[0])
```

Comment:
You have to send the same version as the NCP is using to get it to accept messages. So I try with version 4, and if it reports back with something else, I use that version instead.

I have changed it so that the version function do not take any parameters, and that I'm yielding and not returning.